contents/guides/users/abusive_content.md

```diff
@@ -0,0 +1,20 @@
+---
+title: What can I do if I encounter abusive content or users?
+category: User guides
+---
+
+## Report someone’s post
+
+If you encounter an abusive post, you can click the alert triangle icon at the top right of the post to report it to your podmin. Enter a reason for reporting this post in the dialog box. Please only report posts that break our [community guidelines][community_guidelines] or your pod’s terms of service, for example posts containing illegal content, or which are abusive or spam.
+
+## Ignore a user
+
+If you are currently sharing with that person, removing them from your aspects will stop many of their posts from appearing in your stream. A more complete method is to “ignore” that account. This will prevent any of their posts from appearing in your stream, and they will no longer be able to like or comment on your posts. They will, however, still be able to reshare your posts, comment on reshares of your posts, and their comments on posts by other people which appear in your stream will still be visible to you.
```

Comment:
At the end, " and their comments on other people’s posts will still be visible to you." reads more nicely, but is it clear enough? Do we need to specify "in your stream"?

Reply:
I don't think we need to specify, and "in your stream" is also not true. They'll see comments on other pages as well.